src/app/routes/profile/containers/editprofile.js

```diff
@@ -157,14 +186,14 @@ export default class EditProfile extends React.Component {
                                                 <div className="row">
                                                     <section className="col col-8">
                                                             <label className="input"> <i className="icon-prepend fa fa-phone"/>
-                                                                <input type="tel" name="phone" ref="phone" placeholder="Phone" data-smart-masked-input="(999) 999-9999"/>
+                                                                <input type="tel" name="phone" ref="phone" id="phone" placeholder="Phone" defaultValue={""} />
                                                             </label>
                                                     </section>
                                                     <section className="col col-4">
                                                         <label className="select">
-                                                            <select name="phonetype" ref="phonetype">
-                                                                <option value="" disabled selected>Phone Type</option>
-                                                                {phone_types.map(function(type) {
+                                                            <select name="phonetype" ref="phonetype" defaultValue={""}>
```

Comment:
I would think setting the properties first would be better if we were worried about which properties were set first. All in all, defaultValue will be set from the redux store, and won't necessarily make a difference its its the first property or last